### ISSUE TITLE
Starting Pinapple Docker image fails with: linux spec user: unable to find user pineapple: no matching entries in passwd file

### ISSUE THREAD

**Issue by athrane** [athrane/pineapple#302]
Starting the docker image fails with:

```
<2019-02-02 16:37:06,732> <WARN> <RestTemplate> <POST request for "http://192.168.34.10:8082/containers/alpha01/start" resulted in 500 (Internal Server Error); invoking error handler>
<2019-02-02 16:37:06,733> <ERROR> <RestResponseErrorHandler> <Status code: 500>
<2019-02-02 16:37:06,733> <ERROR> <RestResponseErrorHandler> <Body: {"message":"linux spec user: unable to find user pineapple: no matching entries in passwd file"}
>
<2019-02-02 16:37:06,734> <ERROR> <RestResponseErrorHandler> <Headers: {Api-Version=[1.39], Content-Type=[application/json], Docker-Experimental=[false], Ostype=[linux], Server=[Docker/18.09.1 (linux)], Date=[Sat, 02 Feb 2019 14:55:26 GMT], Content-Length=[97]}>
<2019-02-02 16:37:06,734> <ERROR> <RestResponseErrorHandler> <Message: linux spec user: unable to find user pineapple: no matching entries in passwd file>
<2019-02-02 16:37:06,734> <DEBUG> <ExecutionResultImpl> <Message added with id [Error Message] and content [Failed to to start container with error code [500] and error: linux spec user: unable to find user pineapple: no matching entries in passwd file.].>

```

Used Docker file:
```
# Pull base image for builder image
# ----------------------------------
FROM openjdk:11.0-jdk-oraclelinux7 as builder

RUN set -eux; \
	yum install -y unzip; \
	rm -rf /var/cache/yum

# Install Pineapple binary 
COPY pineapple-standalone-web-client-1.11.0.zip /tmp/archive.zip 
RUN unzip /tmp/archive.zip -d /opt

# Create default user, directory and make pineapple executable
RUN	chmod +x /opt/pineapple-standalone-web-client-1.11.0/bin/setup.sh
RUN	/opt/pineapple-standalone-web-client-1.11.0/bin/setup.sh /opt/pineapple-standalone-web-client-1.11.0 pineapple 

# Temporary solution to make logging work
RUN mkdir -p /var/pineapple/logs 
RUN chown -R $pineapple:$pineapple /var/pineapple
RUN chmod -R 775 /var/pineapple

# Pull base image for final image
# --------------------------------
FROM openjdk:11.0-jdk-oraclelinux7 

# Make pineapple executable
WORKDIR /opt/pineapple-standalone-web-client-1.11.0

# Copy directories from builder
COPY --from=builder /opt/pineapple-standalone-web-client-1.11.0 .
COPY --from=builder /var/pineapple /var/pineapple

# Expose port to host
EXPOSE 8080

# Define volume
VOLUME /var/pineapple

# set user 
USER pineapple

# Start pineapple when container is created
ENTRYPOINT /opt/pineapple-standalone-web-client-1.11.0/runPineapple.sh
```

**Comment by athrane** [comment]
https://jtreminio.com/blog/running-docker-containers-as-current-host-user/

**Comment by athrane** [comment]
Resolved as part of issue #244 where creation of the Pineapple user failed.
